adminapi/src/test/resources/json/datausageinfo.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
You could embed this as a string in tests.

Reply:
I could, but given the string would be pretty large and this project is still supporting Java 8 (text blocks are Java 15+), I would still prefer to have it in resources

Reply:
> One thing that's questionable currently is the usage of `Date` for `DataUsageInfo.lastUpdate` field. While I would want to switch it to newer API (such as `ZonedDateTime` used in `api` module), it would mean introduction of new `com.fasterxml.jackson.datatype:jackson-datatype-jsr310` dependency, so it that's ok, I would include it there as well. Thanks

Please go with `ZonedDateTime`

Reply:
@balamurugana done